package.json

```
@@ -5,8 +5,8 @@
     "main": "localfs.js",
     "scripts": {
         "test": "echo \"Error: no test specified\" && exit 1",
-        "lint": "eslint -c .eslintrc *.js",
-        "lint:fix": "eslint -c .eslintrc *.js --fix"
+        "lint": "eslint -c .eslintrc \"*.js\"",
+        "lint:fix": "eslint -c .eslintrc \"*.js\" --fix"
```

```suggestion
        "lint": "eslint -c .eslintrc \"**/*.js\"",
        "lint:fix": "eslint -c .eslintrc \"**/*.js\" --fix"
```

@ppawlowski I realise there are no sub directories in here, but while we are updating the lint script, should we consider a future where a sub-dir exists in this repo?